src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java

```diff
@@ -8,7 +8,8 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
 
 import com.puppycrawl.tools.checkstyle.checks.imports.*;
-import com.puppycrawl.tools.checkstyle.checks.imports.unusedimports.InputUnusedImportsBug;//violation
```

Comment:
Here as I broke this import statement into two lines and the test file is checking and expecting the whole import statement in the same line and hence, it is giving me an error, how should I fix it?

Reply:
Can you put comment to line 11, as it is expected

Reply:
> Can you revert update of test method, to keep 11 as line of violation and place comment in Input to 11 line, as it is expected

@romani 
I kept everything the same as it was in the last commit but moved the violation comment to line 11 and also changed the line of violation to 11 in the test file instead of 12. and it worked.
Please verify these changes.